blueprints/flexible-topologies.md

```diff
@@ -250,11 +250,29 @@ response JSON:
 
 ##### `/deliveryservices` endpoints
 
-All relevant Delivery Service APIs will have their JSON request and response objects modified to include a new `topology` field which references the name of the topology it's assigned to:
-```JSON
+All relevant Delivery Service APIs will have their JSON request and response objects modified to include the following new fields:
```

Comment:
The same treatment should be applied to rawRemap.  This also lets us remove the ##OVERRIDE anymap hack I think.